### DISCUSSION TITLE
[Question] Where view entities should be located?

### DISCUSSION THREAD

**Discussion by Cododoc** [jasontaylordev/CleanArchitecture#402] [Q&A]
In OnModelCreating method on my DbContext I have this code:

```modelBuilder.Entity<TodoItemView>().HasNoKey().ToView("vReadTodoItems");```

vReadTodoItems - it is SQL view to read some data, that i cant read by linq query.
TodoItemView - wrapper on default TodoItem, that has extra properties.

Is the right place to put my TodoItemView class in Domain layer to Views folder?

**Comment by ShreyasJejurkar** [comment]
You can put those in the Application project instead of Domain. Because it's not a Domain entity as such.

**Comment by jasontaylordev** [comment]
I think it is fine to include these data types in the domain layer. Suggest creating a class to manage configuration, e.g. TodoItemViewConfiguration : IEntityTypeConfiguration<TodoItemView>, instead of using OnModelCreating.
